jcl/src/java.base/share/classes/java/lang/System.java

```diff
@@ -63,9 +63,11 @@
 import com.ibm.jvm.io.ConsolePrintStream;
 /*[ENDIF] PLATFORM-mz31 | PLATFORM-mz64 | !Sidecar18-SE-OpenJ9 */
 
+/*[IF JAVA_SPEC_VERSION >= 17]*/
+import jdk.internal.util.SystemProps;
+/*[ENDIF]*/
```

Comment:
Why did you reorder this rather than have it continue to appear after `Field`?
Please add the missing end comment.